lib/opensusebasetest.pm

```diff
@@ -806,7 +806,7 @@ sub wait_boot_textmode {
     return if check_var('ARCH', 's390x');
 
     my $ready_time       = $args{ready_time};
-    my $textmode_needles = [qw(linux-login emergency-shell emergency-mode)];
+    my $textmode_needles = [qw(tty1-selected emergency-shell emergency-mode)];
```

Comment:
This broke powerVM scenarios and I guess might break even more scenarios. Shall we revert for now?

Reply:
Yes, please, and paste here the affected jobs so we can use to verify a proper solution for all cases.
Thanks!

Reply:
https://openqa.suse.de/tests/3847630#step/first_boot/2 sorry, missed that. Thanks!

Reply:
it looks that here we need a proper needle for ppc64le.

Reply:
Not only a proper needle. The console is called hvc0, so

```perl
my $textmode_needles = [qw(tty1-selected hvc0-selected emergency-shell emergency-mode)];
```